experimental/packages/sdk-logs/test/browser/export/BatchLogRecordProcessor.test.ts

```diff
@@ -24,8 +24,13 @@ import { InMemoryLogRecordExporter } from './../../../src/export/InMemoryLogReco
 const describeDocument =
   typeof document === 'object' ? describe : describe.skip;
 
+/**
+ * VisibilityState has been removed from TypeScript 4.6.0+
+ */
```

Comment:
Should we add a TODO comment here to use `DocumentVisibilityState` later? :thinking:

Reply:
done